GLES2 RENDERER: Error Initializing Shaders

### Please fill out the following:
- **Swayfx Version:**
  - 0.2
- `sway -v`
  - 1.8.1

- **Debug Log:**
https://pastebin.com/WZYSZiLq
Sway running script: https://pastebin.com/zE8D55dR

- **Description:**
  Sway works fine with wlroot-nvidia package. However, SwayFX can't create a renderer.

Relevant lines:

```
00:00:00.095 [ERROR] [wlr] [EGL] command: eglQueryDmaBufModifiersEXT, error: EGL_BAD_PARAMETER (0x300c), message: "EGL_BAD_PARAMETER error: In eglQueryDmaBufModifiersEXT: Invalid format
"
00:00:00.095 [ERROR] [wlr] [render/egl.c:841] Failed to query dmabuf number of modifiers
```

```
00:00:00.168 [ERROR] [sway/desktop/fx_renderer.c:112] Failed to compile shader
00:00:00.168 [ERROR] [sway/desktop/fx_renderer.c:356] GLES2 RENDERER: Error Initializing Shaders
00:00:00.168 [ERROR] [sway/server.c:92] Failed to create fx_renderer
```

found it, the shader was failing with the error `0(60) : error C7502: OpenGL does not allow type suffix 'f' on constant literals in versions below 120`, so i removed the 'f' suffixes i found and it compiled and ran

> found it, the shader was failing with the error `0(60) : error C7502: OpenGL does not allow type suffix 'f' on constant literals in versions below 120`, so i removed the 'f' suffixes i found and it compiled and ran (i hope that didn't break anything, i assume it didn't?). haven't tested under tty yet tho so maybe that's still broken
> edit: yup, also works on tty now.

Thanks a ton for this investigation!

--- a/sway/desktop/shaders/tex.frag
+++ b/sway/desktop/shaders/tex.frag
@@ -56,7 +56,7 @@ void main() {
         vec2 corner_distance = min(gl_FragCoord.xy - position, size + position - gl_FragCoord.xy);
         if (max(corner_distance.x, corner_distance.y) < radius) {
             float d = radius - distance(corner_distance, vec2(radius));
-            float smooth = smoothstep(-1.0f, 0.5f, d);
+            float smooth = smoothstep(-1.0, 0.5, d);
             gl_FragColor = mix(vec4(0), gl_FragColor, smooth);
         }
     }

